src/react-compat.js

```diff
@@ -134,6 +134,8 @@ if (REACT013) {
   };
 }
 
+import SyntheticEvent from 'react/lib/SyntheticEvent';
```

Comment:
nit: please move this to the top of the file. `import` statements get executed before anything else when transpiled by babel anyway (and in the spec), so putting this here can be misleading.